app/models/profile.rb

```diff
@@ -17,6 +17,12 @@ class Profile < ApplicationRecord
 
   delegate :email, to: :user
 
+  scope :search_profiles, (lambda do |search_query|
+    left_outer_joins(:subjects)
```

Comment:
we can use just a `joins` here in order to filter out incomplete profiles (without subjects)